ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.qml

```diff
@@ -65,6 +65,7 @@ Item {
                 Button {
                     id: connectButton
                     text: "Load service"
+                    enabled: model.loading ? false : true;
```

Comment:
if you are binding to an expression that returns bool, then it is redundant to use the ternary operator:

```suggestion
                    enabled: !model.loading
```